TwinePlatform/twine-monolith
# ✨ manage visit categories w/ select

fixes #385 

### Changes
- Add select box to activities table to allow changing category
- Display activities sorted by ID
- Move Activity page related components into own directory
- Add "Saving" beat loader on all write events

### Testing Requirements
- [x] Visitor
  - Add new visit activity
    - Check newly added activity has correct category
  - Update category of existing activity
  - Delete activity
  - Loading spinner should be visible on page even in scrolled down

### Release Requirements
- PO approval

### Manual Deployment
- Visitor app
